### DISCUSSION TITLE
How to enable audit logging on PostgreCluster?

### DISCUSSION THREAD

**Discussion by zenigmadev** [autobase-tech/autobase#721] [Q&A]
Her can i use pgaudit for logging and should i install it on all nodes, i see audit logs on pgadmin but can't under /var/log/postgresql.

**Comment by vitabaks** [comment]
Yes, you can install any extensions by defining the necessary packages in the `postgresql_packages` variable and the necessary options in the `postgresql_parameters` variable if the extension requires it.
